jaggeryapps/admin/templates/header.jag

```diff
@@ -33,15 +33,31 @@ if (userInfo) {
 }
 const USERPORTAL_LINK = "/user-portal/t/" + tenantDomain;
 var headerConfig;
+var identityMgtUrl;
+var mainMenu;
+var userMenu;
+var expandMenu;
+var tenantDisplayName;
+
+headerConfig  = readFile("/conf/toplink_menu.json");
```

Comment:
So every time header gets loaded we read the file. Can you check whether this is pattern followed in other cases also